<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rohini" failures="2" tests="2" name="Test(failed)" time="21.424" errors="0" timestamp="2025-09-14T19:07:05 IST">
  <testcase classname="testCases.TC002_LoginTest" name="verify_LoginTest" time="11.594">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC002_LoginTest.verify_LoginTest(TC002_LoginTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginTest -->
  <testcase classname="testCases.TC001_AccountRegistration" name="AccountRegistration" time="2.947">
    <failure type="java.lang.AssertionError" message="Test failed: expected [Your Account Has Been Created!!] but found [Your Account Has Been Created!]">
      <![CDATA[java.lang.AssertionError: Test failed: expected [Your Account Has Been Created!!] but found [Your Account Has Been Created!]
at testCases.TC001_AccountRegistration.AccountRegistration(TC001_AccountRegistration.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- AccountRegistration -->
</testsuite> <!-- Test(failed) -->
